index.Rmd

```diff
@@ -79,7 +79,7 @@ petty_albums <- get_artist_albums(
 
 ```
 ## Band Specifics
-### Number of Albums Released
+### Number of Albums Released 
 ```{r}
 album_filter <- petty_albums%>%
   filter(name!= c("Mojo Tour Edition", "Hard Promises (Reissue Remastered)", 
```

Comment:
Good use of the filter function, removing those albums definitely helps clarify your data.